src/blocks/services/styles/editor.scss

```diff
@@ -0,0 +1,59 @@
+.wp-block-coblocks-services {
+	[data-block] {
+		margin-bottom: 0;
+		margin-top: 0;
+	}
+
+	.block-editor-inner-blocks {
+		width: 100%;
+	}
+
+	.block-editor-block-list__layout {
+		display: flex;
+		flex-flow: row nowrap;
+		justify-content: space-between;
+
+		// On small screens, we are no longer using row direction but column.
+		@media all and (max-width: 514px) {
+			flex-direction: column;
+		}
+
+		.block-editor-block-list__layout {
+			display: block;
+		}
+	}
+
+	.wp-block[data-type="coblocks/service"] {
+		flex: 1 100%;
+		max-width: 100%;
+
+		> .block-editor-block-list__block-edit {
+			margin-top: 1em;
+			margin-bottom: 1em;
+		}
+	}
+
+	@media all and (min-width: 514px) {
+		> .block-editor-block-list__block-edit {
+			margin-top: 0;
+			margin-bottom: 0;
+		}
+
+		&[data-columns="2"] .wp-block[data-type="coblocks/service"] {
+			flex: 1 calc(100% / 2);
+			max-width: calc(100% / 2);
+		}
+
+		&[data-columns="3"] .wp-block[data-type="coblocks/service"] {
+			flex: 1 calc(100% / 3);
+			max-width: calc(100% / 3);
+		}
+	}
+
+	.wp-block-image .block-editor-rich-text,
```

Comment:
Is this necessary?

Reply:
Nope, this is left over from when I was trying to utilize the image block but eventually just used the image functionality from our Food and Drinks block. It'll be removed.